templates/events/event.html

```diff
@@ -116,9 +116,20 @@ <h5 class="modal-title" id="exampleModalLabel">
 
   <!-- filter and select operations -->
   {% if isUser %}
-  <div class="media">
-    <a class="btn btn-dark" href="{{ url_for('user_events.user_events') }}" role="button">Back To All Events</a>
-  </div>
+      <div>
+          {% if post.superEventID and post.superEventID != '' %}
+              <div class="media" style="float: left; margin-right: 2px">
+                <a class="btn btn-dark" href="{{ url_for('user_events.user_events') }}" role="button">Back To All Events</a>
+              </div>
+              <div class="media">
```

Comment:
Please add some space between the buttons. Thanks.

Reply:
@clav1d there is comments in code and also please fix the confilcts.